### ISSUE TITLE
4.0.0 Beta: 29.05 Encounter search by patient+date fails due to status parameters

### ISSUE THREAD

**Issue by bh021790** [onc-healthit/onc-certification-g10-test-kit#136]
US Core v4.0.0 V.0.2.2 / bottom of webpage: VERSION 0.3.2
Browser: Chrome

I'm looking understand why the Encounter search by patient+date test includes multiple status parameters:

https://fhir-myrecord.cerner.com/r4/17316440-14fe-4957-b62f-9d85de2c88ca/Encounter?date=gt2021-12-07T16:35:11%2B00:00&patient=4755812**&status=planned&status=arrived&status=triaged&status=in-progress&status=onleave&status=finished&status=cancelled&status=entered-in-error&status=unknown**

I'm also curious why this is included with the date: %2B00:00

THANKS!

**Comment by yunwwang** [comment]
16:35:11%2B00:00 is html encoded value for 16:35:11+00:00
I will take a look of the status issue.

**Comment by bh021790** [comment]
Do you have a link to where it calls out the requirement for %2B00:00? We don't support it today.

**Comment by yunwwang** [comment]
http://hl7.org/fhir/us/core/STU4/CapabilityStatement-us-core-server.html#encounter:

[date](http://hl7.org/fhir/us/core/STU4/SearchParameter-us-core-encounter-date.html) (date):
A client SHALL provide a value precise to the second + time offset.
A server SHALL support a value precise to the second + time offset.

**Comment by yunwwang** [comment]
@bh021790 Is this test on inferno.healthit.gov? If so, what is the test session id?

**Comment by bh021790** [comment]
https://inferno.healthit.gov/suites/test_sessions/8942692c-26eb-441f-8d29-31beca302f8c#us_core_v400-us_core_v400_encounter

**Comment by bh021790** [comment]
Yes you are correct - we don't quite handle the offset  properly and require milliseconds  so we are working to resolve that. I just wanted to understand the other search parameters/behavior.

**Comment by yunwwang** [comment]
@bh021790 Though I cannot reproduce the exactly query string, I think the above PR I referenced should fix that problem. We will include it in the next release.

**Comment by bh021790** [comment]
Thankyou!!!

**Comment by jj027509** [comment]
Has the fix for this issue been deployed yet?

**Comment by yunwwang** [comment]
Not yet. PR was merged. The release will be out soon

**Comment by yunwwang** [comment]
@jj027509 @bh021790  US Core Test Kit 0.2.3 is released. Could you test if this release address this issue?

**Comment by jj027509** [comment]
@yunwwang I believe I am still seeing the same issue 
https://inferno.healthit.gov/suites/test_sessions/29bcb43a-05ac-4bc2-96c9-aa586885443e#us_core_v400-us_core_v400_encounter

**Comment by yunwwang** [comment]
@jj027509 
Oops, sorry, I forgot that you tested on inferno.healthit.gov. I thought you had local installation. Then you have to wait for our next deployment.

At the same time, I noticed that the first query (without status) still failed. The server SHALL be able to handle date search value with precision of second+offset according to US Core v4.0.0. Actually, if the first query without status passes, you will not see the second query.
